### PR TITLE
MOTECH-2632 Changes between different openmrs configurations are not saving in tasks.

### PR DESCRIPTION
**Pull request by jablonskidawid**
Added subject to openMRS actions.

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kmadej** [comment]
test this please

File: openmrs/src/main/java/org/motechproject/openmrs/tasks/builder/ActionBuilder.java [hunk 1]
```diff
@@ -81,14 +82,16 @@ private ActionEventRequest buildCreateEncounterAction(String configName) {
         return new ActionEventRequestBuilder()
                 .setDisplayName(getDisplayName(CREATE_ENCOUNTER, configName))
                 .setServiceInterface(OPENMRS_ACTION_PROXY_SERVICE)
-                .setServiceMethod("createEncounter")
+                .setServiceMethod(serviceMethod)
+                .setSubject(serviceMethod + "." + configName)
```
Comment (kmadej) [inline openmrs/src/main/java/org/motechproject/openmrs/tasks/builder/ActionBuilder.java]: please add method for creating subject

**Comment by kmadej** [comment]
test this please

**Pull request merged**